<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COFS-DTS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/HP/Pictures/Camera Roll/20220113093124062FOH.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>961</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="showing">
     <attribute name="title">
      <string>显示页面</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="collect">
     <attribute name="title">
      <string>采集页面</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>441</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>210</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>停止采集</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="settingButton">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>开始采集</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="offlineButton">
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>复位</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="onlineButton">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>340</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>缩放轴</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>20</y>
         <width>81</width>
         <height>77</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_X">
          <property name="text">
           <string>X轴</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Y">
          <property name="text">
           <string>Y轴</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_XY">
          <property name="text">
           <string>XY轴</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>140</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存波形图</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_axis">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>300</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>坐标显示</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_full">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>全屏显示</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_scroll">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>280</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>滚动显示</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="setting">
     <attribute name="title">
      <string>设置页面</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>write_test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>read_test</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="MaintoolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>onlineButton</tabstop>
  <tabstop>offlineButton</tabstop>
  <tabstop>settingButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>verticalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>customPlot</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>system_exit()</slot>
 </slots>
</ui>
